go.mod

```diff
@@ -18,15 +19,15 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/vishvananda/netlink v1.0.0
 	gopkg.in/yaml.v2 v2.3.0
-	k8s.io/api v0.18.4
-	k8s.io/apimachinery v0.18.4
+	k8s.io/api v0.18.6
+	k8s.io/apimachinery v0.18.6
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/kube-proxy v0.0.0-20190918162534-de037b596c1e
 	k8s.io/utils v0.0.0-20200603063816-c1c6865ac451
```

Comment:
could you change it to v.0.18.6 as well ?

Reply:
cannot change k8s.io/utils to v.0.18.6 it doesnt version...I changed it to the version that kube 0.18.6 uses

Reply:
ic, no tag in that repo.